frontend/src/libs/packages/notification/notification.package.ts

```diff
@@ -0,0 +1,36 @@
+import { type ToastOptions } from 'react-toastify';
+import { toast } from 'react-toastify';
+
+class Notification {
+  public error(message: string): void {
+    this.show(message, {
+      type: 'error',
+    });
+  }
+  public info(message: string): void {
+    this.show(message, {
+      type: 'info',
+    });
+  }
+  public success(message: string): void {
+    this.show(message, {
+      type: 'success',
+    });
+  }
+  public warning(message: string): void {
+    this.show(message, {
+      type: 'warning',
+    });
+  }
+  public default(message: string): void {
+    this.show(message, {
+      type: 'default',
+    });
+  }
+
+  private show(message: string, parameters: ToastOptions): void {
```

Comment:
```suggestion
  private show(message: string, parameters: NotificatinOptions): void {
```
create your own type `NotificatinOptions`. For now, it should have only one property `type`